### PR TITLE
Javascript validation

### PR DESCRIPTION
**Pull request by DanielAjibike**

### CODE REVIEW

File: app/uk/gov/hmrc/bforms/views/landfill_tax_form.scala.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <h2>About you</h2>
         </p>
 
-        <form action=@uk.gov.hmrc.bforms.controllers.routes.LandfillTaxForm.landfillTaxForms(registrationNumber:String).url method="POST" class="js-form" autocomplete="off" novalidate="novalidate">
+        <div action=@uk.gov.hmrc.bforms.controllers.routes.LandfillTaxForm.landfillTaxForms(registrationNumber:String).url method="POST" class="js-form" autocomplete="off" novalidate="novalidate">
```
Comment (VlachJosef) [inline app/uk/gov/hmrc/bforms/views/landfill_tax_form.scala.html]: What is the reason for this change?

Comment (DanielAjibike) [inline app/uk/gov/hmrc/bforms/views/landfill_tax_form.scala.html]: I didn't make this change.

Comment (VlachJosef) [inline app/uk/gov/hmrc/bforms/views/landfill_tax_form.scala.html]: Git blame is saying something else. Probably unintended change, would be nice to put `form` back
